ownCloud File Provider UI/Info.plist

```diff
@@ -23,7 +23,24 @@
 	<key>NSExtension</key>
 	<dict>
 		<key>NSExtensionFileProviderActions</key>
-		<array/>
+		<array>
+			<dict>
+				<key>NSExtensionFileProviderActionActivationRule</key>
+				<string>TRUEPREDICATE</string>
+				<key>NSExtensionFileProviderActionIdentifier</key>
+				<string>com.owncloud.FileProviderUI.Share</string>
+				<key>NSExtensionFileProviderActionName</key>
+				<string>Share this Item</string>
```

Comment:
As discussed in the coordination call earlier, this should be renamed to `Share with user/group`.

Reply:
fixed!